### ISSUE TITLE
Whitelist qz protocol handler Google Chrome

### ISSUE THREAD

**Issue by tresf** [qzind/tray#526]
Chrome 77 removed the checkbox that allows whitelisting of a URL handler.  More information here: ~~https://support.google.com/chrome/thread/14194567?msgid=15347344~~
https://support.google.com/chrome/thread/14193532?hl=en

Starting with the `certs-rewrite` branch, we have more granular control over installation steps (they'e done by Java now).

<details>
<summary>Original proposal (changed in comments below)</summary>

Starting with the `certs-rewrite` branch, we have a JSON writer we can use to set this preference as well as a dedicated area for setting up user settings on launch.  We can probably shim this setting in.

**Windows:** `%LOCALAPPDATA%\Google\Chrome\User Data\Default\Preferences`
**Mac:** \<TBD\>
**Linux:** \<TBD\>
```js
"protocol_handler":{"excluded_schemes":{"qz":false}}
```
</details>

**Comment by tresf** [comment]
Found the following recommendation on a [Google Help Forum](https://support.google.com/chrome/thread/14322141?hl=en):


> You can manually whitelist applications/protocol by adding a registry key for it (windows-only)
> 
> Create a txt document with the info below in it and import into the registry. The users will no longer see the prompt. 
> 
> ```reg
> Windows Registry Editor Version 5.00
> [HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Google\Chrome\URLWhitelist]
> "13"="ms-word://*"
> ```

Tested on Windows 10 swapping out `"13"="ms-word://*"`with `"0"="qz://*"` and it's working.  Will work on a patch now.

**Comment by tresf** [comment]
After some digging, found some documentation for Mac (potentially): https://www.chromium.org/administrators/mac-quick-start

```bash
sudo -s
cd /Library/Managed Preferences/<username>
plutil -convert xml1 com.google.Chrome.plist
cat com.google.Chrome.plist
```

**Comment by tresf** [comment]
Support for Windows and MacOS has been added in #582.  Linux will require a bit more investigation.

**Comment by tresf** [comment]
Linux seems to honor the following per: https://github.com/chromium/chromium/blob/b4511255f9592a4aeee1d6d2e10db07e80c157df/chrome/common/chrome_paths.cc#L496-L500

```bash
sudo mkdir -p /etc/opt/chrome/policies/managed
echo '{ "URLWhitelist": ["qz://*"] }' > /etc/opt/chrome/policies/managed/qz-tray.json
echo '{ "URLWhitelist": ["qz://*"] }' > /etc/chromium/policies/managed/qz-tray.json
```

**Comment by tresf** [comment]
This is added in #582 and will be available starting with 2.1.1.  White-label customers have immediate access to this feature.
